site/content/en/docs/concepts/admission_check.md

```diff
@@ -1,23 +1,23 @@
 ---
 title: "Admission Check"
-date: 2023-10-05
+date: 2024-13-06
```

Comment:
@PBundyra  🧐

Reply:
oh hehe, it should be `2024-06-13`

As it is right now, it might prevent this page from rendering.

Reply:
yeah, that is literally my curious about this date :)